srcpkgs/spago/template

```diff
@@ -0,0 +1,29 @@
+# Template file for 'spago'
+pkgname=spago
+version=0.16.0
+revision=1
+build_style="haskell-stack"
```

Comment:
If you define your own `do_build` amd `do_install`, there is no value in using `build_style`, especially since the `haskell-stack` style pulls in the system GHC that you don't use. Just drop this line.